flink-runtime/src/main/java/org/apache/flink/runtime/security/token/DefaultDelegationTokenManagerFactory.java

```diff
@@ -34,37 +31,21 @@
 import java.io.IOException;
 import java.util.concurrent.ExecutorService;
 
-/** A factory for {@link KerberosDelegationTokenManager}. */
+/** A factory for {@link DefaultDelegationTokenManager}. */
 @Internal
-public class KerberosDelegationTokenManagerFactory {
+public class DefaultDelegationTokenManagerFactory {
```

Comment:
Do we need this Factory after we removed the hadoop references?

Reply:
Yes, because I personally don't want to put if conditions to multiple places + this condition can be changed (just like we changed it now).

Reply:
After the simplification this is a bit thin class for my taste, especially for one which is used only in 2 places, but it's your call.